platform/nodejs/index.js

```diff
@@ -182,6 +182,25 @@ async function useLists(lists, options = {}) {
 
 /******************************************************************************/
 
+class MockStorage {
```

Comment:
@gorhill the basic idea is the same as what I did for the Cliqz benchmarks. Afterwards you can refactor the code and remove the need for the mock storage.